src/api-server.ts

```diff
@@ -112,9 +113,10 @@ function subscribe(parameters: any, callback: any): void {
 */
 async function publish(parameters: any, callback: any): Promise<void> {
     //TODO if there's no active stream the server should warn the user
+    const sender = "111" //GET FROM metada
 
     console.log(`publishing ${parameters.request.message.payload} in topic ${parameters.request.topic.channelId} `)
-    const status: any = await publishToRoom(parameters.request.topic.channelId, parameters.request.message.payload);
+    const status: any = await publishToRoom(parameters.request.topic.channelId, parameters.request.message.payload,parameters.request.topic.channelId,sender);
```

Comment:
- This function has 4 parameters, maybe we could wrap them in an object to make it a little more expressive. 
- Why is the parameters.request.topic.channelId twice?